chromeExtension/styles.css

```
@@ -0,0 +1,140 @@
+  font-size: 0.9rem;
+}
+
+button {
+  background: var(--accent);
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  transition: background 0.2s;
+}
+button:hover {
+  background: darken(var(--accent), 10%);
```

Biome (1.9.4)

[error] Unexpected unknown function: darken

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)